### DISCUSSION TITLE
`from anyio import ExceptionGroup` broken on master.

### DISCUSSION THREAD

**Discussion by jonathanslenders** [agronholm/anyio#519] [Q&A] [answered]
Now that we depend on the `exceptiongroup` library, it seems that this top level import is gone. This can break code when upgrading. Would it make sense to expose the `ExceptionGroup` again from the top level `__init__`, or is that something anyio generally doesn't do?

**Comment by agronholm** [comment, marked as answer]
Given that the new exception group is completely API incompatible with the old one, I don't think it makes any sense to re-export that from `anyio.__init__`. Current master is the base for AnyIO 4.0 which comes with other breaking changes so you'll have to adjust your code anyway.

**Comment by jonathanslenders** [comment]
Thank you. That makes sense!
